sickbeard/providers/hdspace.py

```diff
@@ -180,15 +179,15 @@ def login(self):
         login_params = {
             'uid': self.username,
             'pwd': self.password,
-            'page': 'login',
+            'submit': 'Confirm',
         }
 
         response = self.get_url(self.urls['login'], post_data=login_params, returns='text')
         if not response:
             logger.log('Unable to connect to provider', logger.WARNING)
             return False
 
-        if re.search('Password Incorrect', response):
+        if re.search('Automatic log off after 15 minutes inactivity', response):
```

Comment:
Is there a specific reason this is one through regex? If it's a simple string, could as well be 'string' in response right?